swift/vapor/Sources/main.swift

```diff
@@ -0,0 +1,19 @@
+import Vapor
+
+let drop = Droplet(environment: .production)
```

Comment:
On looking over this again I'm pretty sure stating the environment here is unnecessary. As far as I can tell this only applies to multiple config environments being present.